pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+# Data Science PR Template
+
+---
+
+[Link to the PR Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)
+
+---
+
+( BELOW IS AN EXAMPLE )
+
+---
+
+[Related Trello Card](your link here)
+
+---
+

```

Comment:
A little confused about what goes in the 'location' section

Reply:
The 'Location' section is where you put the files that you touched/worked on in which you are doing the pull request for. Does that help to clarify things?